_includes/references.md

```diff
@@ -117,9 +117,9 @@
 [BIP174]: https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki
 
 [@bitcoincoreorg]: https://twitter.com/bitcoincoreorg
-[#bitcoin-core-dev]: https://webchat.freenode.net?channels=%23bitcoin-core-dev
-[#bitcoin-builds]: https://webchat.freenode.net?channels=%23bitcoin-builds
-[##bitcoin-core-gui]: https://webchat.freenode.net?channels=%23%23bitcoin-core-gui
+[#bitcoin-core-dev]: https://libera.chat/
+[#bitcoin-builds]: https://libera.chat/
+[#bitcoin-core-gui]: https://libera.chat/
```

Comment:
```suggestion
[#bitcoin-core-dev]: https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#bitcoin-core-dev
[#bitcoin-builds]: https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#bitcoin-builds
[#bitcoin-core-gui]: https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#bitcoin-core-gui
```

Docs: https://github.com/kiwiirc/kiwiirc/wiki/Linking-to-and-Embedding-Kiwi-IRC

Going to send to each channel a test message from `hebasto_kiwi" user.

Reply:
Messages have been sent successfully.

Reply:
Thank you for testing! Will update